src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -71,6 +71,19 @@ public static function fromString(string $cookie, bool $decode = false)
         return new static($name, $value, $data['expires'], $data['path'], $data['domain'], $data['secure'], $data['httponly'], $data['raw'], $data['samesite']);
     }
 
+    /**
+     * @param string                        $name     The name of the cookie
+     * @param string|null                   $value    The value of the cookie
+     * @param int|string|\DateTimeInterface $expire   The time the cookie expires
+     * @param string                        $path     The path on the server in which the cookie will be available on
+     * @param string|null                   $domain   The domain that the cookie is available to
+     * @param bool|null                     $secure   Whether the client should send back the cookie only over HTTPS or null to auto-enable this when the request is already using HTTPS
+     * @param bool                          $httpOnly Whether the cookie will be made accessible only through the HTTP protocol
+     * @param bool                          $raw      Whether the cookie value should be sent with no url encoding
+     * @param self::SAMESITE_*|''|null      $sameSite Whether the cookie will be available for cross-site requests
```

Comment:
we try to remove docblock when they don't add much value, and we do partial `@param` when only some arguments benefit from the docblock. Please remove the lines that are obvious here.

Reply:
now I see that `__construct` has the same docblock, so maybe we considered that there is enough value in keep it :)

Reply:
I think the CS fixer kept it due to having a description.